gson/src/main/java/com/google/gson/JsonSyntaxException.java

```diff
@@ -26,18 +25,29 @@
 
   private static final long serialVersionUID = 1L;
 
+  /**
+   * Creates exception with the specified message. If you are wrapping another exception, consider
+   * using {@link #JsonSyntaxException(String, Throwable)} instead.
+   *
+   * @param msg error message describing a possible cause of this exception.
+   */
   public JsonSyntaxException(String msg) {
     super(msg);
   }
 
+  /**
+   * Creates exception with the specified message and cause.
+   *
+   * @param msg   error message describing what happened.
+   * @param cause root exception that caused this exception to be thrown.
+   */
   public JsonSyntaxException(String msg, Throwable cause) {
     super(msg, cause);
   }
 
   /**
-   * Creates exception with the specified cause. Consider using
-   * {@link #JsonSyntaxException(String, Throwable)} instead if you can
-   * describe what actually happened.
+   * Creates exception with the specified cause. Consider using {@link #JsonSyntaxException(String,
+   * Throwable)} instead if you can describe what happened.
```

Comment:
Only changes where the line is wrapped (and "actually" has been removed), so change is probably not needed